app/client/templates/autoform/imageUpload.js

```diff
@@ -1,34 +1,35 @@
+var $hiddenInput,
+$fileInput,
+$thumbnail;
+
 Template.imageUpload.rendered = function () {
-  // TODO: Declare new Slingshot uploader
-  // Session.set('uploader', new Slingshot.Upload("imageUpload"));
 };
 
 Template.imageUpload.events({
   'change .image-upload-input': function(e) {
     e.preventDefault();
 
-    // TODO: Upload file
-    /*
-       $hiddenInput = $('#' + this.atts.id);
-       $loader = $hiddenInput.siblings('.preloader-wrapper');
-       $thumbnail = $hiddenInput.siblings('.image-upload-thumbnail');
-
-       $loader.addClass('active');
-
-       uploader.send($hiddenInput.siblings('.image-upload-input')[0].files[0], function (error, downloadUrl) {
-       if (error) {
-       console.error('Error uploading', error.reason);
-       Materialize.toast(TAPi18n.__('alert-error'), 2000);
-       Materialize.toast(error.reason, 3000);
-       } else {
-       $hiddenInput.val(downloadUrl);
-       $thumbnail.attr('src', downloadUrl).show();
-       Materialize.toast(TAPi18n.__('alert-upload_success'), 3000);
-       }
-
-       $loader.removeClass('active');
-       });
-       */
+    var uploader = new Slingshot.Upload("imageUpload");
```

Comment:
this reclares a new uploader on every change to the input. Is that good? Maybe part of the issues?